docs/services/navigation/_index.md

```diff
@@ -224,15 +269,35 @@ Get the current location of the robot in the navigation service.
 - [(*geo.Point)](https://pkg.go.dev/github.com/kellydunn/golang-geo#Point): The current location of the robot in the navigation service, represented in a `Point` with latitude (lat) and longitude (lng) values.
 - [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
 
-For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk@v0.2.49/services/navigation#Service).
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/navigation#Service).
 
 ```go
-myNav, err := navigation.FromRobot(robot, "my_nav")
+myNav, err := navigation.FromRobot(robot, "my_nav_service")
 
 // Get the current location of the robot in the navigation service
 location, err := myNav.Location(context.Background(), nil)
 ```
 
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(navigation.GeoPoint)](https://python.viam.dev/autoapi/viam/services/navigation/index.html#viam.services.navigation.GeoPoint): The current location of the robot in the navigation service, represented in a `GeoPoint` with latitude (lat) and longitude (lng) values.
```

Comment:
Hmm, where did you get `lat` and `lng` from? They appear to be the full words according to the linked SDK reference entry. 

EDIT: Ah, I see earlier references to these names, betting they changed at some point separate from this code change.